modules/visual_script/visual_script_property_selector.cpp

```diff
@@ -542,7 +553,7 @@ void VisualScriptPropertySelector::_notification(int p_what) {
 	}
 }
 
-void VisualScriptPropertySelector::select_method_from_base_type(const String &p_base, const String &p_current, bool p_virtuals_only) {
+void VisualScriptPropertySelector::select_method_from_base_type(const String &p_base, const String &p_current /*= ""*/, const bool p_virtuals_only /*= false*/, const bool p_connecting /*= true*/) {
```

Comment:
I don't think it's a good idea to add default arguments as comments. We don't do it anywhere else in the code base, it looks weird/makes the signature harder to read, and they're bound to get out of sync if the default value were to change.

Let the IDEs take care of presenting such information, if the C++ language doesn't allow it in the first place.

Reply:
I understand, updated.